Symfony/CS/Fixer/Contrib/ClassElementsOrderFixer.php

```diff
@@ -0,0 +1,305 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Gregor Harlan <gharlan@web.de>
+ */
+final class ClassElementsOrderFixer extends AbstractFixer
+{
+    private static $types = array(
+        'use_trait' => null,
+        'const' => null,
+        'public' => null,
+        'protected' => null,
+        'private' => null,
+        'property' => null,
+        'property_static' => array('property'),
+        'property_public' => array('property', 'public'),
+        'property_protected' => array('property', 'protected'),
+        'property_private' => array('property', 'private'),
+        'property_public_static' => array('property_static', 'property_public'),
+        'property_protected_static' => array('property_static', 'property_protected'),
+        'property_private_static' => array('property_static', 'property_private'),
+        'method' => null,
+        'method_static' => array('method'),
+        'method_public' => array('method', 'public'),
+        'method_protected' => array('method', 'protected'),
+        'method_private' => array('method', 'private'),
+        'method_public_static' => array('method_static', 'method_public'),
+        'method_protected_static' => array('method_static', 'method_protected'),
+        'method_private_static' => array('method_static', 'method_private'),
+        'construct' => null,
+        'phpunit' => null,
```

Comment:
> But which one would be first, second etc?

order of these methods will not be changed. they are only moved to the desired place.